### PR TITLE
Create student index and show API endpoints

### PR DESCRIPTION
**Pull request by jwashke**
* Create users table
* Create students table
* Create cohort table
* Create students index endpoint
* Create students show endpoint
* Add specs for all of the above
* Create seed file for sample data

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,3 +17,66 @@ This is the back end api for the Turing MentorSHIP project. It is written in Rai
...
+  "student": {
+    "id": 1,
+    "github_avatar":  "https://avatars2.githubusercontent.com/u/14855129?v=3&s=460",
+    "name": "student 0",
+    "email": "student0@turing.io",
+    "phone_number": "555-555-5555",
+    "slack_username": "student0",
+    "cohort": 1602
+  }
+}
```
Comment (notmarkmiranda) [inline README.md]: nice documentation for the api response!

**Comment by kristindiannefoss** [comment]
Wow, your commits are really thorough and your code is super clean.  Really organized.  This is like looking at a surgical log.  It must be really easy to follow your progress.  I should make about 85% more small commits.

**Comment by jwashke** [comment]
Thanks @kristindiannefoss, I've been trying to focus on better commits lately.

**Comment by jwashke** [comment]
Closing this PR as it is deprecated by PR #2

**Pull request closed without merging**